pkgs/development/python-modules/pysonic/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi, pythonOlder }:
+
+buildPythonPackage rec {
+  pname = "py-sonic";
+  version = "0.7.7";
+  disabled = pythonOlder "3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "4cea42a2b0dc2ed0fd8568d6bf0509cfa2675a8b1c347ce9364a00881ebc0272";
+  };
+
+  doCheck = false;
```

Comment:
If the upstream project has no tests add a comment to indicate that.

If there are no tests you should at least add a `pythonImportsCheck`, see https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck.

Reply:
added `pythonImportsCheck` and a comment